modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -31,41 +31,70 @@ endif::[]
 
 . Stop all of the machine configuration pools managed by the Machine Config Operator (MCO):
 
-** Stop the master configuration pool:
+** Stop the `master` configuration pool:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch MachineConfigPool master --type='merge' --patch \
   '{ "spec": { "paused": true } }'
 ----
 
-** Stop the worker machine configuration pool:
+** Stop the `worker` machine configuration pool:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch MachineConfigPool worker --type='merge' --patch \
   '{ "spec":{ "paused": true } }'
 ----
 
 . To prepare for the migration, set the migration field to `null` by entering the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": null } }'
 ----
 
-. To start the migration, set the network plugin back to OpenShift SDN by entering the following commands:
+. Apply the patch to the `Network.operator.openshift.io` object to set the network plugin back to OpenShift SDN by entering the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": { "networkType": "OpenShiftSDN" } } }'
+----
+
+. Confirm that the `Network.operator.openshift.io cluster` object's network plugin is `OpenShiftSDN` by entering the following command:
```

Comment:
I think we may have to clarify, if the desired output is not `OpenShiftSDN`, wait for couple of seconds?

Reply:
If the command output does not show OpenShiftSDN, wait a few seconds for the patch operation to complete.

Reply:
Thanks, @Rupesh-git-eng . Updated. Please add a LGTM if all is technically accurate? Thanks!